### DISCUSSION TITLE
初学者前来报到

### DISCUSSION THREAD

**Discussion by pipeng119** [haixiangyan/jest-tutorial#33] [Ideas]
技术栈: Angular
公司要求写unit test
刚开了开头，觉得非常不错~！

**Comment by pipeng119** [comment]
看完mock网页地址这一章
你永远可以相信npm
这是否也是人口红利的另一种提现？
(=^_^=)

**Reply by haixiangyan** [reply]
个人喜欢 all in one 的东西，不想自己思考了~

**Comment by ycshill** [comment]
我也来打卡了，最近小组正好搞单测
